### DISCUSSION TITLE
<BezierEdge/> not applying marker ends or animation

### DISCUSSION THREAD

**Discussion by TobiasKrok** [xyflow/xyflow#2419] [General]
Hi!

I've made a custom edge using `<BezierEdge/>` but for some reason it's not applying the marker or animations. Am I doing something wrong here?

Here's a sandbox reproducing the issue: https://codesandbox.io/s/prod-tree-d0fjoi?file=/src/CustomEdge.js

**Comment by moklick** [comment]
Hey @TobiasKrok 

you need to define `animated` and `markerEnd` at the edge object in the `edges` array. Another way is to use `defaultEdgeOptions`:

```jsx
// outside of the component
const defaultEdgeOptions = {
  animated: true,
  markerEnd: {
    type: MarkerType.ArrowClosed
  }
};

// ...

return (
  <ReactFlow defaultEdgeOptions={defaultEdgeOptions} />
);
```

**Reply by TobiasKrok** [reply]
Thank you, that fixed the marker issue! However, I want to animate the edge when you click. Is this possible through the props passed to CustomEdge? F.ex `animated={props.selected}`
